### ISSUE TITLE
Money Sinks: Gambling

### ISSUE THREAD

**Issue by LightOracle** [StrangeLoopGames/EcoSuggestions#1231]
I had a player on White Tiger suggest a cool idea for a money sink to help economy keep flowing. Gambling for cool items would be a way to spend money you don't know what to do with it. If there was a way to have the government set the prizes or if the developers came up with a system that would reward players for spending their money.

As an example, you could spin a wheel to see if you get a steam truck without having to play top dollar for the ones on the current market. You cold gamble to get an Eco statue, only end up with 50 limestone, or you could walk away with a skid steer. The more ways players have to spend money on items they may want keeps money from being horded.

Players spend money on things they find value in or have to buy because they can't make them. The more options that are out there the more money can flow so there is a constantly supply and demand. A lot of items probably also need more costs to them and deterioration. I don't think taxes fix all these issues and this is a system that might keep players playing and new players coming into an established server.

**Comment by RoseFlunder** [comment]
I would be also down for a Roulette Table where the owner could link the account like at a store.
Government run roulette casino and the earnings the government makes with it could for example go into a county road funding project or reforesting project or whatever.

**Comment by LightOracle** [comment]
Something I did think about is that kids play this game. So if it could be an idea but not for this reason it would have to be on an adult only server I guess.

**Comment by ThePiachu** [comment]
It's not really a money sink - money doesn't get removed. Plus this sort of gambling is regressive - the poor have more incentives to gamble (to get the cool stuff they want), while the rich either are the bank or can buy whatever the bank already has, so they won't gamble as much. So unless you're gambling away some say, important land claims, rights, etc. it's not helping the economy, it drains it.

**Comment by LightOracle** [comment]
> It's not really a money sink - money doesn't get removed. Plus this sort of gambling is regressive - the poor have more incentives to gamble (to get the cool stuff they want), while the rich either are the bank or can buy whatever the bank already has, so they won't gamble as much. So unless you're gambling away some say, important land claims, rights, etc. it's not helping the economy, it drains it.

You are right. I didn't think about it that way. I do know that it would be a fun activity for someone to spend money on if there were say things you could buy that you couldn't get in game. However, it still is a topic I would like to find a solution on. Some way of getting money to flow from people to gov to people again or a reason to spend money. Right now the economy just ends up with people sitting on it. What good does it do to sit on money you are not going to spend? The servers last usually 15-30 days. Then you loose it all anyway. I don't want money to actually leave the economy but to circulate better. Perhaps there is just a need for a lot more specialties that have valuable goods to buy from.

**Comment by ThePiachu** [comment]
You could tax people based on how much money they have, but then they would just buy stuff to have less money and hoard goods. Would be interesting to see some "wealth tax" based on the goods you have, but that would probably be fiddly.

Alternatively, inflate the problem away - print money and give it to the people to spend. Enough to keep the economy flowing, not enough to make an impact before day 30 hopefully. Combine both of them and you can have a stable money system.

You probably also want some late game resource sinks - golden statues, TVs, modern living stuff to get that housing up and those skills flowing for ultra rich to dump resources into. Seems food and housing are the only two things everyone wants late game, once they have the basic vehicles...

**Comment by LightOracle** [comment]
Everyone always thinks taxes are the answer when time and time again they have proven to not work. People need incentive to spend their money taxes just make them hoard more. The point of money is to spend it on stuff you want or need. Make more things for them to want or need then you have money flowing. This means something to sink their money into. The issue with that being housing point items is there is no reason to have SP once you reach all the skills you want. There needs to be some kind of system that continues to let you progress or have the feeling of progression. You can't stop those who just want to horde money to see how much they can obtain. To them that is the goal or endgame for them. Others just want it for a way to get the things they care about. To some that is endgame building mats to make cool houses even if they don't need the SP. To others it's decoration or to get better equipment. I think what we need is a way to sink money into maybe a bonus item you can use on your next cycle or something worth the effort. Perhaps just requiring more things to be needed for more professions could be the answer.
